recipes/geogram/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class GeogramConan(ConanFile):
+    name = "geogram"
+    description = "a programming library with geometric algorithms"
+    license = "BSD 3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/BrunoLevy/geogram"
+    topics = ("graphics programming", "mesh generation", "geometry processing", "graphics libraries", "mesh processing")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_graphics": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_graphics": False,
+    }
+
+    short_paths = True
+
+    @property
+    def _min_cppstd(self):
+        return 11
```

Comment:
Are you sure this project requires C++11? Running grep I only found test, example and third party using C++11/14/17

Reply:
then by default I should put "98"?

Reply:
Just skip and don't check anything. We usually add it to avoid breaking with new compilers.